### ISSUE TITLE
Problem when cell_type_labels are numeric

### ISSUE THREAD

**Issue by MostBeef** [digitalcytometry/cytospace#52]
Thanks for creating this tool.
I have now successfully run cytospace on my own dataset. 

I had an issue which I managed to solve but thought I would leave a comment in case anyone has a similar issue.

I have a scRNA dataset where clusters name have yet to be renamed . I.e. the names that were carried through into the cell_type_labels were integers corresponding to the cluster number in the original scRNAseq analysis. 

```
cell_fraction_estimates.txt
Index	0	1	10	11	12	2	3	4	5	6	7	8	9
Fraction	0.13884869	0.10281999	0.15212866	0.08104075	0.01107279	0.06391414	0.09356885	0.12036684	0.04423891	0.07535366	0.03214314	0.04633949	0.03816410
```

Cytospace threw the following error when running:

```
Traceback (most recent call last):
  File "/data/home/hfx857/.conda/envs/cytospace/bin/cytospace", line 8, in <module>
    sys.exit(run_cytospace())
  File "/data/home/hfx857/.conda/envs/cytospace/lib/python3.8/site-packages/cytospace/cytospace.py", line 382, in run_cytospace
    main_cytospace(**arguments)
  File "/data/home/hfx857/.conda/envs/cytospace/lib/python3.8/site-packages/cytospace/cytospace.py", line 357, in main_cytospace
    solve_linear_assignment_problem(scRNA_data, st_data, cell_type_data,
  File "/data/home/hfx857/.conda/envs/cytospace/lib/python3.8/site-packages/cytospace/cytospace.py", line 90, in solve_linear_assignment_problem
    calculate_cost(scRNA_data, st_data, cell_type_data, cell_type_numbers_int,
  File "/data/home/hfx857/.conda/envs/cytospace/lib/python3.8/site-packages/cytospace/linear_assignment_solvers/linear_assignment_solvers.py", line 58, in calculate_cost
    unique_cell_type_labels = sorted(np.unique(cell_type_labels.values[:,0]), key=str.lower)
TypeError: descriptor 'lower' for 'str' objects doesn't apply to a 'numpy.int32' object 
```

Relabeling the cell type labels (and the corresponding labels in the cell type label file) solved the issue and cytospace has run successfully. 

```
cell_fraction_estimates.txt
Index	CellType0	CellType1	CellType10	CellType11	CellType12	CellType2	CellType3	CellType4	CellType5	CellType6	CellType7	CellType8	CellType9
Fraction	0.13884869	0.10281999	0.15212866	0.08104075	0.01107279	0.06391414	0.09356885	0.12036684	0.04423891	0.07535366	0.03214314	0.04633949	0.03816410
```
I'm guessing the underlying cause is that when the labels only contain numerals they are assigned to a numpy.int32 instead of a str variable.

**Comment by erinlbrown** [comment]
Thank you for noting this! We will update the code to prevent this type of error.

**Comment by hsjeon-k** [comment]
Thank you for letting us know about the issue! We will address this error in the next update.

**Comment by hsjeon-k** [comment]
Hello!

The issue with the numeric column names should have been resolved with yesterday's update (v1.0.2).
Thank you for suggesting a workaround in the meantime, and please let us know if a similar issue arises!

Best,
Hyun Soo
